src/funcConfig/local.settings.ts

```diff
@@ -17,16 +23,43 @@ export interface ILocalSettingsJson {
     ConnectionStrings?: { [key: string]: string };
 }
 
-export const azureWebJobsStorageKey: string = 'AzureWebJobsStorage';
-
-export async function getAzureWebJobsStorage(context: IActionContext, projectPath: string): Promise<string | undefined> {
+export async function getLocalConnectionString(context: IActionContext, connectionKey: ConnectionKeyValues, projectPath?: string): Promise<string | undefined> {
     // func cli uses environment variable if it's defined on the machine, so no need to prompt
-    if (process.env[azureWebJobsStorageKey]) {
-        return process.env[azureWebJobsStorageKey];
+    if (process.env[connectionKey]) {
+        return process.env[connectionKey];
+    }
+
+    projectPath ??= getWorkspaceRootPath();
+    if (!projectPath) {
+        return undefined;
     }
 
     const settings: ILocalSettingsJson = await getLocalSettingsJson(context, path.join(projectPath, localSettingsFileName));
-    return settings.Values && settings.Values[azureWebJobsStorageKey];
+    return settings.Values && settings.Values[connectionKey];
+}
+
+export async function validateStorageConnection(context: IActionContext, options?: Omit<IValidateConnectionOptions, 'suppressSkipForNow'>, projectPath?: string): Promise<void> {
```

Comment:
Where will this be used? I noticed that it's not anywhere right now (not even in the commented out code)

Reply:
It will be used during deploy I believe, it wouldn't be in this PR yet